### PR TITLE
update job submission docs

### PR DESCRIPTION
**Pull request by hzy46**
According to the user interviews, the following contents have been changed:

1. Changed the tutorial link on the main page.
2. Emphasized the SSH feature explanation, and created a page for building images with SSH.

### CODE REVIEW

File: docs/user/build_docker_with_ssh.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build Docker Images with SSH
```
Comment (scarlett2018) [inline docs/user/build_docker_with_ssh.md]: can you merge this with /job_docker_env.md?

File: docs/user/job_submission.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ The Hello World job is set to download the CIFAR-10 dataset and train a simple m
 
   If an appropriate Docker image isn't found, you could [build a Docker image](../job_docker_env.md) by your self.
 
-  Note, if a Docker image doesn't include *openssh-server* and *curl* packages, it cannot use SSH feature of OpenPAI. If SSH is needed, a new Docker image can be built and includes *openssh-server* and *curl* on top of the existing Docker image.
+  Note, **if a Docker image doesn't include *openssh-server* and *curl* packages, it cannot use SSH feature of OpenPAI**. If SSH is needed, a new Docker image can be built and includes *openssh-server* and *curl* on top of the existing Docker image. Please refer to [this tutorial](build_docker_with_ssh.md) for details.
```
Comment (scarlett2018) [inline docs/user/job_submission.md]: **Important note**: if you'd like to ssh to the docker within OpenPAI, make sure *openssh-server* and *curl* packages are included by the docker image.

File: docs/user/build_docker_with_ssh.md (hunk 1 above)
Comment (hzy46) [reply docs/user/build_docker_with_ssh.md]: I added a section in job_docker_env.md. Please check it [here](https://github.com/microsoft/pai/blob/b5f17fb7516b064a36d7db99bb07de415833bd08/docs/job_docker_env.md#enable-ssh-for-your-image).

Comment (hzy46) [reply docs/user/build_docker_with_ssh.md]: I added a section in job_docker_env.md. Please check it [here](https://github.com/microsoft/pai/blob/b5f17fb7516b064a36d7db99bb07de415833bd08/docs/job_docker_env.md#enable-ssh-for-your-image).

File: docs/user/job_submission.md (hunk 2 above)
Comment (hzy46) [reply docs/user/job_submission.md]: have resolved

File: docs/job_docker_env.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# Use docker to package the job environment dependencies
+# Use Docker to Package the Job Environment Dependencies
 
-The system launches a deep learning job in one or more Docker containers. A Docker images is required in advance. 
+The system launches a deep learning job in one or more Docker containers. A Docker images is required in advance.
```
Comment (debuggy) [inline docs/job_docker_env.md]: image

File: docs/job_docker_env.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 The system provides a base Docker images with HDFS, CUDA and cuDNN support, based on which users can build their own custom Docker images.
```
Comment (debuggy) [inline docs/job_docker_env.md]: image

Comment (hzy46) [reply docs/job_docker_env.md]: fixed :)

**Pull request merged**